packages/graphiql/vite.config.ts

```diff
@@ -7,9 +7,6 @@ export default defineConfig({
   plugins: [reactRefresh()],
   server: {
     port: 4001,
-    proxy: {
-      '/graphql': 'http://localhost:4000',
-    },
```

Comment:
Why remove the proxy? How do you test stuff in development?

Reply:
We just send the HTTP request directly to the server(It was helpful while testing CORS)

Reply:
See https://github.com/dotansimha/graphql-yoga/pull/895/files#diff-a18c0ae4dc9c71e0e5d6c8c4a45263acc431bca9847b3de23f9a3dc355f1cc21R25